<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogViewerWidget</class>
 <widget class="QWidget" name="LogViewerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>936</width>
    <height>448</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Log viewer</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QGridLayout" name="logFileAndLevelGridLayout">
     <property name="horizontalSpacing">
      <number>20</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="logFileLabel">
       <property name="text">
        <string>Log &amp;file:</string>
       </property>
       <property name="buddy">
        <cstring>logFileComboBox</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="logLevelComboBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>130</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>130</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="logLevelLabel">
       <property name="text">
        <string>Log &amp;level:</string>
       </property>
       <property name="buddy">
        <cstring>logLevelComboBox</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QComboBox" name="logFileComboBox">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>220</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <spacer name="logFileAndLevelHorizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="filterByContentVerticalLayout">
     <item>
      <widget class="QLabel" name="filterByContentLabel">
       <property name="text">
        <string>Filter by &amp;content:</string>
       </property>
       <property name="buddy">
        <cstring>filterByContentLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="filterByContentLineEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="logEntriesVerticalLayout">
     <item>
      <widget class="QLabel" name="logEntriesLabel">
       <property name="text">
        <string>Log &amp;entries:</string>
       </property>
       <property name="buddy">
        <cstring>logEntriesTableView</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableView" name="logEntriesTableView"/>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <layout class="QVBoxLayout" name="filterByLogLevelVerticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>&amp;Filter by log level:</string>
       </property>
       <property name="buddy">
        <cstring>filterByLogLevelTableWidget</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="filterByLogLevelTableWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>220</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="leftButtonsHorizontalLayout">
     <item>
      <widget class="QPushButton" name="clearPushButton">
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="resetPushButton">
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="leftButtonsHorizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="copyAllToClipboardPushButton">
       <property name="text">
        <string>Copy all to clipboard</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveToFilePushButton">
       <property name="text">
        <string>Save to file...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="1">
    <layout class="QHBoxLayout" name="tracePushButtonHorizontalLayout">
     <item>
      <spacer name="tracePushButtonHorizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Ignored</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>100</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="tracePushButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Trace</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
